test/http2/server.go

```diff
@@ -37,7 +40,13 @@ func Serve() {
 		port := lookupEnv("HTTP_PORT", defaultHTTPPort)
 		log.Printf("Listening on port %v\n", port)
 
-		if err := http.ListenAndServe(":"+port, nil); err != nil {
+		h2s := &http2.Server{}
+		h1s := &http.Server{
```

Comment:
nit: is this http1? Or still http2 but with cleartext?

Reply:
It's cleartext HTTP/1.  I'll add comments to clarify.